rudder-inventory-ldap/SOURCES/patches/openldap/0001-patch-memory-leak-replication-7735.patch

```diff
@@ -0,0 +1,67 @@
+Subject: [PATCH] ITS#7735 plug memleak
+
+Introduced in commit 48cdd54d9accb9283807ed592369b72654fb302d
```

Comment:
This is useful, but please also document the commits (0645878d5dc4c08b7fd8d5bf39a166d74650f15b and ed092229634baf8c5a9f554ea664a2468d6f5a6a) you cherry-picked to make this patch, and include a link to the ITS issue (http://www.openldap.org/its/index.cgi/Software%20Bugs?id=7735;selectid=7735).

This will greatly simplify maintenance of this patch in the future.